.github/workflows/stackhpc-all-in-one.yml

```diff
@@ -329,15 +329,16 @@ jobs:
             -v $(pwd):/stack/kayobe-automation-env/src/kayobe-config \
             -e KAYOBE_ENVIRONMENT -e KAYOBE_VAULT_PASSWORD -e KAYOBE_AUTOMATION_SSH_PRIVATE_KEY \
             ${{ steps.kayobe_image.outputs.kayobe_image }} \
-            /stack/kayobe-automation-env/src/kayobe-config/.automation/pipeline/playbook-run.sh etc/kayobe/ansible/migrate-rabbitmq-queues.yml --tags rabbit-queue-migration
+            /stack/kayobe-automation-env/src/kayobe-config/.automation/pipeline/script-run.sh tools/rabbitmq-quorum-migration.sh --skip-checks
```

Comment:
Skip checks? :(

Reply:
Well the checks used to fail you see :slightly_smiling_face:

Reply:
I think they'll pass now. I'll remove the skip once the rabbit changes merge in Antelope so I can actually test the changes here